README-2.md

```
@@ -0,0 +1,864 @@
+# <p align="center"><img src="images/cypilot-kit.png" alt="Cypilot Banner" width="100%" /></p>
+
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](LICENSE)
+[![Version](https://img.shields.io/badge/version-3.6-green.svg)]()
```

markdownlint-cli2 (0.22.0)

[warning] No empty links

(MD042, no-empty-links)